### ISSUE TITLE
How did you train your pre-trained model?

### ISSUE THREAD

**Issue by hanbongyoo** [ronghuaiyang/arcface-pytorch#94]
Hello~ I'm a beginner in face recognition.

I wanted to train the model myself to get similar performance to your pre-trained model.
But the problem was that training accuracy was always 0 and threshold was 0.99.
So I used Insightface(arcface_torch) code to train the model myself.
However, despite using a larger dataset(ms1mv3), the performance was worse than your model.

I tried to change Insightface model like your model, but the performance was still worse than your model.(e.g. image channel(3->1), resnet18 model structure, batchsize)
I tested lfw dataset and my own dataset(Asian).
I think the cause of the difference in performance is the learning rate and training epoch.
In Insightface, training epoch is 25 and only 3 decay_epoch.

Can you explain how did you train your model in detail?
or what do you think that the reason about can't catch up your model performance?

Thank you for reading.

**Comment by jonilaserson** [comment]
Also asking the same question.

I was able to achieve 97.46% accuracy on lfw test using this code after 110 epochs.
However, the pretrained model gets 99.33% accuracy.
